src/components/DatePicker/DatePicker.types.ts

```diff
@@ -0,0 +1,14 @@
+export type DateVariant = 'start' | 'end';
+
+export type MinMaxDate = 'min' | 'max';
```

Comment:
you can create enums for DateVariant and MinMaxDate and use them

Reply:
done